.github/workflows/gating.yaml

```diff
@@ -46,3 +46,67 @@ jobs:
               echo 'RUN python3 src/ipa-tuura/manage.py test'
           } > podmanfile
           podman build --tag gatingtest -f ./podmanfile
+
+  build-fedora-ipa-server-container-on-ubuntu:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sssd repository
+        uses: actions/checkout@v3
+        with:
+          path: sssd
+      - name: Setup containers
+        uses: SSSD/sssd-ci-containers/actions/setup@master
+        with:
+          path: sssd-ci-containers
+          tag: latest
+          override: |
+            services:
+              client:
+                image: quay.io/sssd/ci-client-devel:latest
+                shm_size: 4G
+                tmpfs:
+                - /dev/shm
+                volumes:
+                - ../sssd:/sssd:rw
+              keycloak:
+                image: quay.io/ftrivino/keycloak
+                shm_size: 4G
+                tmpfs:
+                - /dev/shm
+                volumes:
+                - ../sssd:/sssd:rw
+
+      - name: Get source
+        uses: actions/checkout@v3
+        with:
+          path: 'ipa-tuura'
+# yamllint disable rule:line-length
+      - name: Create container and run tests
+        run: |
+          {
+              echo 'FROM fedora:38'
+              echo '# set TZ to ensure the test using timestamp'
+              echo 'ENV TZ=Europe/Madrid'
+              echo 'RUN dnf -y update'
+              echo 'RUN dnf -y install python3-sssdconfig maven unzip python3-pip git python3-netifaces python3-devel krb5-devel gcc sssd-dbus dbus-devel glibc glib2-devel dbus-daemon python-devel openldap-devel python-ipalib'
+              echo 'RUN dnf clean all'
+              echo 'COPY ipa-tuura ipa-tuura'
+              echo 'WORKDIR /ipa-tuura'
+              echo 'RUN pip install -r src/install/requirements.txt'
+              echo 'RUN pip install dbus-python python-ldap'
+              echo 'RUN python3 src/ipa-tuura/manage.py test'
+          } > podmanfile
+          podman build --tag gatingtest -f ./podmanfile
```

Comment:
In general it is a Dockerfile or to be consistent Containerfile. It is not crucial just a cosmeting comment